sdk/appcenter-storage/src/main/java/com/microsoft/appcenter/storage/models/DocumentError.java

```diff
@@ -16,7 +18,7 @@
      * @param exception that occurred during the network state change.
      */
     @SuppressWarnings("WeakerAccess")
-    public DocumentError(Throwable exception) {
+    public DocumentError(@NonNull Throwable exception) {
```

Comment:
Do not use support annotations in models or public APIs. We don't depend on those in jCenter artifacts and it can cause weird warnings for customers not having the dependency.